doc/manuals/orion-api.md

```diff
@@ -1,127 +1,81 @@
+            - [Replace all entity attributes [PUT /v2/entities/{entityId}/attrs]](#replace-all-entity-attributes-put-v2entitiesentityidattrs)
+            - [Remove Entity [DELETE /v2/entities/{entityId}]](#remove-entity-delete-v2entitiesentityid)
+        - [Attributes](#attributes)
+            - [Get attribute data [GET /v2/entities/{entityId}/attrs/{attrName}]](#get-attribute-data-get-v2entitiesentityidattrsattrname)
+            - [Update Attribute Data [PUT /v2/entities/{entityId}/attrs/{attrName}]](#update-attribute-data-put-v2entitiesentityidattrsattrname)
+            - [Remove a Single Attribute [DELETE /v2/entities/{entityId}/attrs/{attrName}]](#remove-a-single-attribute-delete-v2entitiesentityidattrsattrname)
+        - [Attribute Value](#attribute-value)
+            - [Get Attribute Value [GET /v2/entities/{entityId}/attrs/{attrName}/value]](#get-attribute-value-get-v2entitiesentityidattrsattrnamevalue)
+            - [Update Attribute Value [PUT /v2/entities/{entityId}/attrs/{attrName}/value]](#update-attribute-value-put-v2entitiesentityidattrsattrnamevalue)
+        - [Types](#types)
+            - [List Entity Types [GET /v2/types/]](#list-entity-types-get-v2types)
+            - [Retrieve entity information for a given type [GET /v2/types/]](#retrieve-entity-information-for-a-given-type-get-v2types)
+    - [Subscriptions Operations](#subscriptions-operations)
+        - [Subscription List](#subscription-list)
+            - [List Subscriptions [GET /v2/subscriptions]](#list-subscriptions-get-v2subscriptions)
+            - [Create Subscription [POST /v2/subscriptions]](#create-subscription-post-v2subscriptions)
+        - [Subscription By ID](#subscription-by-id)
+            - [Retrieve Subscription [GET /v2/subscriptions/{subscriptionId}]](#retrieve-subscription-get-v2subscriptionssubscriptionid)
+            - [Update Subscription [PATCH /v2/subscriptions/{subscriptionId}]](#update-subscription-patch-v2subscriptionssubscriptionid)
+            - [Delete subscription [DELETE /v2/subscriptions/{subscriptionId}]](#delete-subscription-delete-v2subscriptionssubscriptionid)
+    - [Registration Operations](#registration-operations)
+        - [Registration list](#registration-list)
+            - [List Registrations [GET /v2/registrations]](#list-registrations-get-v2registrations)
+            - [Create Registration [POST /v2/registrations]](#create-registration-post-v2registrations)
+        - [Registration By ID](#registration-by-id)
+            - [Retrieve Registration [GET /v2/registrations/{registrationId}]](#retrieve-registration-get-v2registrationsregistrationid)
+            - [Update Registration [PATCH /v2/registrations/{registrationId}]](#update-registration-patch-v2registrationsregistrationid)
+            - [Delete Registration [DELETE /v2/registrations/{registrationId}]](#delete-registration-delete-v2registrationsregistrationid)
+    - [Batch Operations](#batch-operations)
```

Comment:
This renders this way:

![imagen](https://user-images.githubusercontent.com/1534240/171370228-52201c7a-5f8e-4896-ae51-772fde3fe959.png)

So something weird is happening here...

Reply:
The problem behind this render problem is there is a "step" title leves between Batch Operations (##) and the next API method, Update [POST /v2/op/update] (####). There are, at least, 2 possible solutions.
1.- Increase title level for API operation (from #### to ###)
2.- Include new section between 2 titles, as an example:
```
## Batch Operations

### Update operations

#### Update [POST /v2/op/update]
[...]
```

The second alternative is the one that makes more sense because of homogeneity of the sections (API methods, with the proper route are always 4 level titles (####)

Reply:
Done in https://github.com/telefonicaid/fiware-orion/pull/4142/commits/d85f33f5089a5e6c5c98cc8f17ea014f1e650564